docs/autogen.py

```diff
@@ -53,19 +53,15 @@ def get_function_signature(function, method=True):
         args = args[: -len(defaults)]
     else:
         kwargs = []
-    st = "%s.%s(" % (clean_module_name(function.__module__), function.__name__)
+    st = ["%s.%s(" % (clean_module_name(function.__module__), function.__name__)]
 
     for a in args:
-        st += str(a) + ", "
+        st.append(str(a))
     for a, v in kwargs:
```

Comment:
maybe a bit more intuitive names than `a`, `v`  and `st`?

Reply:
I change `a` to `arg`, `v` to `value` and `st` to `signature` 😃

Reply:
I guess the `a` in kwargs should be `key`?